app/profile/[[...username]]/page.tsx

```diff
@@ -43,15 +51,22 @@ const Profile = async ({ params }: { params: { username?: string[] } }) => {
     ? params.username[0]
     : await getCurrentUsername();
 
-  const user = await getUser();
+  const profile = await getUserProfile(username);
+
+  const followers = profile.followers ? profile.followers.total : 0;
+
   const data = await getData(username);
 
   return (
     <div>
-      <PageTitle
-        title={`Profile — ${username}`}
-        subtitle={`${formatFollowers(user.totalFollowers)} followers`}
-      />
+      <PageTitle title="Profile" />
+      <div className="flex flex-row">
+        {/* @ts-expect-error Server Component */}
+        <ProfileImg username={username} />
```

Comment:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/7117993/221242940-43ccadc3-d7e8-4eb1-94fb-c779f0e69805.png">
can you make the pic bigger? Maybe so that it takes up two lines of text instead of 1?

Reply:
If you think this change should be a different PR that's fine